stackrox/stackrox
# ROX-23678: Delete frontend code for VulnMgmt Reports 1.0

## Description

### Problem

`DeliveryDestination` type was a compromise between Vulnerability Reports 1.0 and 2.0 payload/response types.

### Solution

Delete obsolete files as a prerequisite to make `DeliveryDestination` type consistent with Vulnerability Reports 2.0 and Compliance Reports 2.0 payload/response.

### Residue

1. Decide whether `getDayList` function is still useful in dateUtils.ts file.
2. Take another look at use of reportConstants.ts file.
3. Understand pro and con of separate schedule components in Vulnerability Reports 2.0 and Compliance Reports 2.0

## Checklist

## Testing Performed

Even webpack was not able to shake out all of the unreachable code.

1. `yarn lint` in ui
2. `yarn build` in ui and then compute branch - master for the following
    * `wc build/static/js/*.js`
        main.js -976 = 4633769 - 4634745
        total -63710 = 11532323 - 11596033
    * `ls -al build/static/js/*.js | wc`
        files -3 = 169 - 172
3. `yarn start` in ui

### Manual testing

1. Visit /main/vulnerabilities/reports click **Create report**, and then create a report.
    * See POST /v2/reports/configurations request.
    * See GET /v2/reports/configurations request.
    * See row in table.
